build/docker/zap-baseline.py

```diff
@@ -120,7 +135,7 @@ def main(argv):
     usage()
     sys.exit(3)
     
-  if len(config) > 0 or len(generate) > 0 or len(report) > 0:
+  if len(config_file) > 0 or len(config_url) > 0 or len(generate) > 0 or len(report) > 0:
```

Comment:
Ah yea, it should be report_html and report_xml (not config_url) as they are both written to the mapped drive